server/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "buddy-app-graphql",
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "npm run build:live --runInBand",
+    "build:live": "nodemon --watch 'src/**/*.js' --exec node --inspect src/index.js",
+    "prisma": "prisma"
+  },
+  "dependencies": {
+    "bcryptjs": "^2.4.3",
+    "dotenv": "^8.1.0",
+    "graphql-yoga": "^1.7.0",
+    "jsonwebtoken": "^8.2.0",
+    "prisma-client-lib": "^1.31.0"
+  },
+  "devDependencies": {
+    "nodemon": "^1.19.3",
+    "prisma": "^1.31.0"
```

Comment:
Wonder if it should it be in dev deps and `"prisma-client-lib": "^1.31.0"` is not? 🤔